CompuCell3D/core/post_install_tasks/windows/scriptSetup.py

```diff
@@ -1,171 +1,25 @@
-# data=file.read()
-# file.close()
-# outFile.write(data)
-# os.remove(x)
-
-# outFile.close()
-
-
-# Modifying compucell3d.bat run script 
-tmpFile = open("tmp.bat", "w")
-tmpFile.write("%s\n" % ("@ECHO OFF"))
-tmpFile.write("%s\n" % ("@SET PREFIX_CC3D=" + PREFIX_CC3D))
-tmpFile.write("%s\n" % ("@SET PYTHON_INSTALL_PATH=" + PYTHON_INSTALL_PATH))
-tmpFile.write("%s\n" % r"@SET PATH=%PYTHON_INSTALL_PATH%\Library\bin;%PATH%")
-tmpFile.close()
-
-# concatenate
-outFile = open("compucell3d.bat", "w")
-
-# fileList to concatenate
-fileList = ["tmp.bat", "compucell3d.bat.in.v2"]
-
-# print "CWD=",os.getcwd()
-
-for x in fileList:
-    print("Processing ", x)
-    file = open(x, 'r')
-    data = file.read()
-    file.close()
-    outFile.write(data)
-    os.remove(x)
-
-outFile.close()
-
-# Modifying runScript.bat
-tmpFile = open("tmp.bat", "w")
-tmpFile.write("%s\n" % ("@ECHO OFF"))
-tmpFile.write("%s\n" % ("@SET PREFIX_CC3D=" + PREFIX_CC3D))
-tmpFile.write("%s\n" % ("@SET PYTHON_INSTALL_PATH=" + PYTHON_INSTALL_PATH))
-tmpFile.write("%s\n" % r"@SET PATH=%PYTHON_INSTALL_PATH%\Library\bin;%PATH%")
-tmpFile.close()
-
-# concatenate
-outFile = open("runScript.bat", "w")
-
-# fileList to concatenate
-fileList = ["tmp.bat", "runScript.bat.in.v2"]
-
-# print "CWD=",os.getcwd()
-
-for x in fileList:
-    print("Processing ", x)
-    file = open(x, 'r')
-    data = file.read()
-    file.close()
-    outFile.write(data)
-    os.remove(x)
-
-outFile.close()
-
-# Modifying paramScan.bat script
-tmpFile = open("tmp.bat", "w")
-tmpFile.write("%s\n" % ("@ECHO OFF"))
-tmpFile.write("%s\n" % ("@SET PREFIX_CC3D=" + PREFIX_CC3D))
-tmpFile.write("%s\n" % ("@SET PYTHON_INSTALL_PATH=" + PYTHON_INSTALL_PATH))
-tmpFile.write("%s\n" % r"@SET PATH=%PYTHON_INSTALL_PATH%\Library\bin;%PATH%")
-tmpFile.close()
-
-# concatenate
-outFile = open("paramScan.bat", "w")
-
-# fileList to concatenate
-fileList = ["tmp.bat", "paramScan.bat.in.v2"]
-
-# print "CWD=",os.getcwd()
-
-for x in fileList:
-    print("Processing ", x)
-    file = open(x, 'r')
-    data = file.read()
-    file.close()
-    outFile.write(data)
-    os.remove(x)
-
-outFile.close()
-
-# Modifying twedit++ run script
-tmpFile = open("tmp.bat", "w")
-tmpFile.write("%s\n" % ("@ECHO OFF"))
-tmpFile.write("%s\n" % ("@SET PREFIX_CC3D=" + PREFIX_CC3D))
-tmpFile.write("%s\n" % ("@SET PYTHON_INSTALL_PATH=" + PYTHON_INSTALL_PATH))
-tmpFile.write("%s\n" % r"@SET PATH=%PYTHON_INSTALL_PATH%\Library\bin;%PATH%")
-tmpFile.close()
-
-# concatenate
-outFile = open("twedit++.bat", "w")
-
-# fileList to concatenate
-fileList = ["tmp.bat", "twedit++.bat.in.v2"]
-
-# print "CWD=",os.getcwd()
-
-for x in fileList:
-    print("Processing ", x)
-    file = open(x, 'r')
-    data = file.read()
-    file.close()
-    outFile.write(data)
-    os.remove(x)
-
-outFile.close()
-
-# Modifying celldraw run script
-
-import os.path
-
-if os.path.isfile("celldraw.bat.in.v2"):
-
-    # Modifying pifgenenrator file
+def localize_script(name: str):
```

Comment:
great change. I dont know how many times I looked this previous script and wanted to refactor it but I always found something else to work on :)